src/components/ATATSummaryCard.vue

```diff
@@ -11,118 +11,125 @@
         >
       </v-card-text>
     </v-card>
-    <v-card
-      v-show="data.cards.length > 0"
-      width="40rem"
-      class="v-card ma-9 ml-0 body"
-      v-for="(card, index) in data.cards"
-      :key="index"
-    >
-      <v-card-title class="d-flex justify-space-between align-start pa-6 pb-5">
-        <div>
-          <div
-            class="type body"
-            v-if="card.type && card.type !== 'APPLICATION'"
+    <v-row align="stretch">
+      <v-col cols="4" v-for="(card, index) in data.cards" :key="index">
+        <v-card
+          v-show="data.cards.length > 0"
+          :width="`${cardWidth}rem`"
+          class="v-card ma-9 ml-0 body"
+        >
+          <v-card-title
+            class="d-flex justify-space-between align-start pa-6 pb-5"
           >
-            {{ card.type }}
-          </div>
-          <div class="d-flex align-start">
-            <v-btn
-              :id="'header_link_' + index"
-              :ripple="false"
-              @click="leftButtonClicked(card)"
-              small
-              class="h3 link-button no-focus-shift pa-0"
-              :class="{ 'no-border': card.type === 'APPLICATION' }"
+            <div>
+              <div
+                class="type body"
+                v-if="card.type && card.type == 'TASK ORDER'"
+              >
+                {{ card.type }}
+              </div>
+              <div class="d-flex align-start">
+                <v-btn
+                  :id="'header_link_' + index"
+                  :ripple="false"
+                  @click="leftButtonClicked(card)"
+                  small
+                  class="h3 link-button no-focus-shift pa-0"
+                  :class="{ 'no-border': card.type === 'APPLICATION' }"
+                >
+                  <div class="font-weight-bold">
+                    <div>{{ card.title }}</div>
+                  </div>
+                </v-btn>
+                <v-btn
+                  small
+                  :ripple="false"
+                  @click="leftButtonClicked(card)"
+                  class="
+                    no-focus-shift
+                    pa-0
+                    no-text-decoration no-border
+                    link-button
+                    mx-n2
+                  "
+                  ><v-icon v-if="card.showChevronRight" x-large>
```

Comment:
start of `<v-icon` on same line because whitespace issue if on next line? Or can this drop down a line?

Reply:
this is a product of `es-lint`

Reply:
ok